packages/vue/src/components/confirm-reset-password.vue

```diff
@@ -50,15 +51,8 @@ const onConfirmResetPasswordSubmit = (e: Event): void => {
 };
 
 const submit = (e: Event): void => {
-  const formData = new FormData(<HTMLFormElement>e.target);
-  send({
-    type: 'SUBMIT',
-    //@ts-ignore
-    data: {
-      //@ts-ignore
-      ...Object.fromEntries(formData),
-    },
-  });
+  e.preventDefault();
```

Comment:
You don't need `e.preventDefault()` since it's stoped earlier in the form with `@submit.prevent`